lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -530,29 +735,32 @@ class MouseAdapter extends BaseAdapter {
         print(event.type);
       }
       _callback(_convertWheelEventToPointerData(event));
+      // Prevent default so mouse wheel event doesn't get converted to
+      // a scroll event that semantic nodes would process.
       event.preventDefault();
     });
   }
 
-  List<ui.PointerData> _convertEventToPointerData(
-    ui.PointerChange change,
-    html.MouseEvent event,
-  ) {
-    List<ui.PointerData> data = <ui.PointerData>[];
+  void _convertEventToPointerData({
+    @required List<ui.PointerData> data,
+    @required html.MouseEvent event,
+    @required _SanitizedDetails details,
+  }) {
```

Comment:
Suggestion: if you change this method to take `List<_SanitizedDetails>` and do the for loop here, you could avoid many for loops in the rest of the code.

Reply:
Changed, thanks.